yarn.lock

```diff
@@ -1374,7 +1374,7 @@
     simple-cbor "^0.4.1"
```

Comment:
I'm not sure we need .lock file in repop

Reply:
I've removed from PR this file, thank you